### PR TITLE
DAOS-6645 test: Add more online extend (More tests)

### PR DESCRIPTION
**Pull request by rpadma2**
Test-tag-hw-medium: pr,hw,medium,ib2 online_extend
Skip-unit-tests: true
Skip-func-hw-test-small: true
Skip-func-hw-test-large: true

Signed-off-by: rpadma2 <ravindran.padmanabhan@intel.com>

### CODE REVIEW

File: src/tests/ftest/osa/osa_online_extend.py [hunk 1]
```diff
@@ -56,18 +52,23 @@ def daos_racer_thread(self):
...
-    def run_online_extend_test(self, num_pool, racer=False):
+    def run_online_extend_test(self, num_pool, racer=False,
+                               oclass=None, app_name="ior"):
         """Run the Online extend without data.
             Args:
-             int : total pools to create for testing purposes.
+             num_pool(int) : total pools to create for testing purposes.
+             racer(bool) : Run the testing along with daos_racer.
+                           Defaults to False.
+             oclass(str) : Object Class (eg: RP_2G1, etc)
```
Comment (shimizukko) [inline src/tests/ftest/osa/osa_online_extend.py:62]:
```suggestion
             oclass(str) : Object Class (eg: RP_2G1, etc). Defaults to None.
```

**Review by shimizukko — DISMISSED** [review]

File: src/tests/ftest/osa/osa_online_extend.py (hunk 1 above)
Comment (rpadma2) [reply src/tests/ftest/osa/osa_online_extend.py:62]: Done.

File: src/tests/ftest/osa/osa_online_extend.py [hunk 2]
```diff
@@ -28,23 +28,19 @@ def setUp(self):
...
         self.dmg_command = self.get_dmg_command()
-        self.ior_flags = self.params.get("ior_flags", '/run/ior/iorflags/*')
-        self.ior_apis = self.params.get("ior_api", '/run/ior/iorflags/*')
-        self.ior_test_sequence = self.params.get("ior_test_sequence",
-                                                 '/run/ior/iorflags/*')
-        self.ior_daos_oclass = self.params.get("obj_class",
-                                               '/run/ior/iorflags/*')
-        self.ior_dfs_oclass = self.params.get(
-            "obj_class", '/run/ior/iorflags/*')
+        self.daos_command = DaosCommand(self.bin)
```
Comment (saurabhtandan) [inline src/tests/ftest/osa/osa_online_extend.py:31]: you can use self.get_daos_command() here, as you did for dmg above this line.

**Review by saurabhtandan — APPROVED** [review]
One minor suggestion, otherwise looks good.

File: src/tests/ftest/osa/osa_online_extend.py (hunk 2 above)
Comment (rpadma2) [reply src/tests/ftest/osa/osa_online_extend.py:31]: Yes, Will do it when I re-enable the tests. Thanks.

**Review by shimizukko — APPROVED** [review]
I don't see you enabling any test. Maybe the title is incorrect?

**Comment by rpadma2** [comment]
> I don't see you enabling any test. Maybe the title is incorrect?

Hi Makito, I added new tests... But, we noticed all tests are failing due to two tickets (one created as part of this PR:  @skipForTicket("DAOS-7195,DAOS-7955"))

**Comment by sylviachanoiyee** [comment]
PR looks good.
@rpadma2 , could you update both skipfor tickets with instructions on how to re-enable test at the top of the description?

**Pull request merged**